src/main/java/Lemon.java

```diff
@@ -0,0 +1,156 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Lemon {
+    /*** Messages ***/
+    String logoMsg = "____________________________________________________________\n"
+            + " Hello! I'm Lemon\n"
+            + " What can I do for you?\n";
+    String endMsg = " Bye. Hope to see you again soon!\n"
+            + "____________________________________________________________\n";
+    String barMsg = "____________________________________________________________";
+    String emptyListMsg = " Sowwy, theres currently no tasks in your list.\n Ill be happy to add some for you OwO!";
+    String listMsg = " Here are the tasks in your list:";
+    String markMsg = " Nice! I've marked this task as done:";
+    String unmarkMsg = " OK, I've marked this task as not done yet:";
+    String addTaskMsg = " Got it. I've added this task:";
+    String deleteTaskMsg = " Noted. I've removed this task:";
+
+    /*** Initialising ***/
+    ArrayList<Task> tasks = new ArrayList<>();
+    int numTasks = 0;
+
+    enum Commands {
+        BYE,
+        LIST,
+        MARK,
+        UNMARK,
+        TODO,
+        EVENT,
+        DEADLINE,
+        DELETE
+    }
+    
+    public static void main(String[] args) {
```

Comment:
Could refactor the main function so that your chatbot has a singular run function that controls when it runs.